src/main/java/com/strandls/observation/es/util/ObservationListPageMapper.java

```diff
@@ -458,6 +461,7 @@ public ObservationListPageMapper() {
 	 * @param userGroup
 	 * @param customField
 	 * @param tags
+	 * @param placeName
```

Comment:
remove this unnecesaary comments, Place name is not present as the property of the class